streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -17,40 +17,34 @@
 
 package org.apache.kafka.streams.processor.internals.assignment;
 
+
 import java.util.HashSet;
 import java.util.Set;
 
 public class ClientState<T> {
```

Comment:
@guozhangwang @dguy I did have a quick look at the tests using `ClientState<Integer>` -- I actually do not see any reason why those tests need to use `Integer` instead of `TaskId`. It might be a slight simplification for the test, but if we don't need the generic type in the actual code, a test does not justify to have a generic IMHO and I have a strong opinion about removing it. This would be a nice "beginner" PR so not much work for us. But I want your opinion before I create a JIRA for it.